exopy_hqc_legacy/instruments/drivers/visa/tabor_awg.py

```diff
@@ -249,13 +249,13 @@ def sampling_frequency(self, value):
     def running(self):
         """Run state getter method
         """
-        return '2 : Intrument is running'
+        return True
 
-    @running.setter
     @secure_communication()
-    def running(self, value):
```

Comment:
What is the rational for moving away from a property ?

Reply:
I needed to add a parameter to the setter (delay). If you think that I should leave the default setter alone, I can do that.

Reply:
I missed the use of that parameter, it is fine.